CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # Releases
 
-## Unreleased
+## v1.6.0
 
+## Added
 * Added support for entity namespaces
 * Added support for datasets
 * unstable: Added support for otel-metrics-ingestion
-* Promote gcp-metrics-integration to stable
+* Promotes `unstable/config/GcpMetricsIntegration` to `v1/config/GcpMetricsIntegration`
```

Comment:
can we reword this in terms of `chronoctl` behavior?  does a command change? if not then we can probably just put it as Internal

Reply:
Ope let me repush this with the latest change. Unstable was not released, so effectively we're just adding this to the primary command list.